js/get_help_now.js

```diff
@@ -1,26 +1,42 @@
+/**
+ * showGetHelpActionSection() displays the page section
```

Comment:
As discussed in earlier Pull requests and on slack, we are following AirBnB's style guide for comments. Like php comments, we are mentioning function's name here as well. Reference: [Airbnb style guide](https://github.com/airbnb/javascript#comments)